src/sentry/api/endpoints/internal_beacon.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from django.conf import settings
+from rest_framework.response import Response
+
+from sentry import get_version, options
+from sentry.api.base import Endpoint
+from sentry.http import safe_urlopen
+from sentry.tasks.beacon import BEACON_URL
+
+logger = logging.getLogger("beacon")
+
+
+class InternalBeaconEndpoint(Endpoint):
+    permission_classes = ()
+
+    def post(self, request):
+        error = None
+        install_id = options.get("sentry:install-id")
+
+        if not settings.SENTRY_BEACON:
```

Comment:
```suggestion
        if settings.SENTRY_BEACON is False:
```

The beacon defaults to on, and not sure if we explicitly set this to `True`.

Reply:
This is the same logic as in the worker: https://github.com/getsentry/sentry/blob/54060f0a9a8924922c99ddc786aa3209d0aedfc3/src/sentry/tasks/beacon.py#L37-L38